components/atoms/Avatar/avatar-hover-card.tsx

```diff
@@ -5,37 +5,32 @@ import Link from "next/link";
 import HoverCardWrapper from "components/molecules/HoverCardWrapper/hover-card-wrapper";
 
 import { getAvatarByUsername } from "lib/utils/github";
-import roundedImage from "lib/utils/roundedImages";
```

Comment:
Is this no longer being used in the app? Are we still caching using cloudinary?

Reply:
Yes!, the cloudinary rounded image transformation helper is no longer in use.